docs/ReadMe.md

```diff
@@ -0,0 +1,51 @@
+# Using the MAGE Web SDK
+
+## Integrating client-configuration
+
+Assuming you have bootstrapped a MAGE server application, you need to make sure that the MAGE client-configuration ends
+up in your build. The configuration for the browser application can be extracted from the `mage` object, by calling its
+`mage.getClientConfig(appName)` method. How you integrate the returned object into your build is up to you.
+
+- You can write the configuration to a JSON file that your build system picks up and embeds.
+- You can host the JSON on an HTTP endpoint.
+- You can have both client-code and server-code in a single repository, and integrate the build system into it.
+
+Which approach you take will depend on your desired workflow. In the [integrations](./integrations) folder we provide
+examples for specific build systems.
+
+## Setting up the MAGE Web SDK runtime
+
+Once you have integrated the MAGE Web SDK, you need to register any modules you wish to use with MAGE, and run their
+asynchronous setup methods. It is important to note that for some modules you will probably want to delay their setup
+phase until after a user has been logged in. Because MAGE allows you to register a module *at any time* and allows you
```

Comment:
You may know these answers, but let me start by answering them here (for the record):

- Why? Because authentication is likely required to read from a database, etc.
- Where? After a login operation has called back.
- The only other cases where load order matters is when modules have inter-dependencies, but that would be a userland problem.